### DISCUSSION TITLE
support for MinotaurX and curvehash

### DISCUSSION THREAD

**Discussion by magecrimson** [coinfoundry/miningcore#1344] [General]
Any plans to add support for MinotaurX or curvehash?

**Comment by totom9000** [comment]
Both is suported we have both on mining4people.com

**Comment by magecrimson** [comment]
You may have added it yourself.  The algos are not included in the current version of the miningcore git master or dev.

Sent from my iPhone

On Aug 11, 2022, at 12:42 PM, totom9000 ***@***.***> wrote:

﻿

Both is suported we have both on mining4people.com

**Reply by totom9000** [reply]
they are added our self yes
